ui-components/src/page_components/listing/ListingCard.tsx

```diff
@@ -27,8 +29,10 @@ export interface ListingCardContentProps {
   tableHeader?: ListingCardHeader
   tableSubheader?: ListingCardHeader
 }
+
 export interface ListingCardProps {
   cardTags?: ImageTag[]
+  homeType: HomeTypeEnum
```

Comment:
We're trying to remove all / most business logic from the component library, so passing this in as a prop named `homeType` isn't recommended - we may have other jurisdictions that have no concept of `homeType` in their data model. If possible I think it makes sense to pass this tag into the existing `cardTags` prop as the first item in the array if it exists. It may require a new custom type on top of `ImageTag` that adds a optional color styling.

Reply:
Thanks for tip 😄. Added hometype tag as first in cardTags, and added additional prop for shade